### PR TITLE
206 TaskRunner runSolve()

### PR DESCRIPTION
**Pull request by Anastasiia3795**

### CODE REVIEW

File: src/main/java/org/codewars/utils/TaskRunner.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import java.math.BigInteger;
 
 public class TaskRunner {
+    private static final Reader reader = new Reader();
```
Comment (mosvvy) [inline src/main/java/org/codewars/utils/TaskRunner.java:6]: all methods are static.

Comment (Anastasiia3795) [reply src/main/java/org/codewars/utils/TaskRunner.java:6]: In class TaskRunner, right?

File: src/main/java/org/codewars/utils/TaskRunner.java [hunk 2]
```diff
@@ -94,8 +96,11 @@ public static int[] runSquareOrSquareRoot(){
         return new int[0];
     }
 
-    public static double runSolve(){
-        return 0.0;
+    public void runSolve(){
+        System.out.println("Run task solve\nEnter the sum:");
```
[the remark this answers is not in the document]
Comment (Anastasiia3795) [reply src/main/java/org/codewars/utils/TaskRunner.java:100]: Okay

File: src/main/java/org/codewars/utils/TaskRunner.java (hunk 1 above)
Comment (mosvvy) [reply src/main/java/org/codewars/utils/TaskRunner.java:6]: in Reader i mean

Comment (mosvvy) [reply src/main/java/org/codewars/utils/TaskRunner.java:6]: image
example

**Review by mosvvy — APPROVED** [review]

**Review by RostislavKrotenko — APPROVED** [review]

**Review by mosvvy — DISMISSED** [review]

**Pull request merged**